R/pkg/tests/fulltests/test_context.R

```diff
@@ -21,10 +21,11 @@ test_that("Check masked functions", {
   # Check that we are not masking any new function from base, stats, testthat unexpectedly
   # NOTE: We should avoid adding entries to *namesOfMaskedCompletely* as masked functions make it
   # hard for users to use base R functions. Please check when in doubt.
-  namesOfMaskedCompletely <- c("cov", "filter", "sample", "not")
+  namesOfMaskedCompletely <- c("cov", "filter", "sample", "not", "getFunction")
```

Comment:
to pass `test_context`, I need to mask the build-in [`getFunction`](https://www.rdocumentation.org/packages/methods/versions/3.6.2/topics/methodUtilities) , is this OK? @HyukjinKwon

Reply:
Should probably we should define this with `setMethod` (see `functions.R`) to avoid the complete masking.

Reply:
Ah .. even with that the signature would likely conflict. hm

Reply:
Let's use a different name for now. e.g., `getFunc` and explain it in the docs.

Reply:
got it, thanks. let me update the pr.

Reply:
cc @falaki @felixcheung @shivaram FYI